src/components/area/AreaEditor.vue

```diff
@@ -354,6 +355,15 @@ function onOverCell({ row, col }) {
   overArea.value = props.area
 }
 
+function onOverGridCell({ row, col }) {
+  onOverCell({ row, col })
+  if (overArea.value.display === 'grid') overArea.value.grid.hover = { row, col }
```

Comment:
We should change this to be something like `overArea: { area, cell: { row, coll } }`. If you check the other hover or focused states, they don't modify the design state because that is included in undo/redo automatically. So all this focus info needs to be out of the `mainArea` state tree.